projects/agent-test-harness.md

```diff
@@ -0,0 +1,101 @@
+3. Facilitating Innovation and Adoption:  By providing developers with a tool to validate their agents with those developed by others in the space, Agent Test harness lowers the barriers to entry for creating new, compliant, and interoperable decentralized identity solutions. This facilitates innovation and encourages the adoption of decentralized identity technologies.
+
+## Code of Conduct
+
+ACA-Py follows the [Hyperledger Code of Conduct](https://github.com/hyperledger/aries-agent-test-harness/blob/main/CODE_OF_CONDUCT.md), which is what the OWF code of conduct is based on.
+
+## TAC Sponsor
+
+* Tracy Kuhrt
+* TBD
+
+## Project License
+
+The project has an Apache 2.0 license: [Agent Test Harness - License](https://github.com/hyperledger/aries-agent-test-harness/blob/main/LICENSE)
+
+## Source Control
+
+The project is hosted in GitHub and includes the following repositories:
+
+* [Agent Test Harness](https://github.com/hyperledger/aries-agent-test-harness) — test results published to [https://aries-interop.info](https://aries-interop.info)
+* [Aries Akrida](https://github.com/hyperledger/aries-akrida) — load testing engine for deployments of ACA-Py and other compatible agents.
+
+## Issue Tracker
+
+Issues are tracked using GitHub's Issues feature in the corresponding repository.
+
+## External Dependencies
+
+The Agent Test Harness itself is built in [Python](https://www.python.org/) using the [Behave](https://github.com/behave/behave) [behavior-driven development](https://en.wikipedia.org/wiki/Behavior-driven_development) (BDD) engine. The repository also contains a component per agent implementation that uses Agent Test Harness.
+
+Test results from the periodic (thrice weekly) interoperability test runs are pushed for later analysis into an instance (operated by [Government of British Columbia](https://www2.gov.bc.ca/gov/content/home)) of the [Allure](https://allurereport.org/docs/pytestbdd/) behavior driven development reporting tool. Those results are in turn published to [https://aries-interop.info](https://aries-interop.info) via a GitHub action in the Agent Test Harness repository.
+
+Akrida is built on [Locust](https://locust.io/) and [Credo-TS](https://github.com/openwallet-foundation/credo-ts) to generate loads applied to any deployment.
+
+## Release Methodology
+
+The only “release” from Agent Test Harness is the test execution results summary published here: [https://aries-interop.info](https://aries-interop.info)
+
+## Initial Maintainers
+
+| Name            | Github      | Organization                             |
+| --------------- | ----------- | ---------------------------------------- |
+| Sheldon Regular | nodlesh     | Funded by Government of British Columbia |
+| Stephen Curran  | swcurran    | Funded by Government of British Columbia |
+| Kim Ebert       | KimEbert42  | Indicio, PBC                             |
```

Comment:
I believe we should include Alex Walker | anwalker293 | Inidicio, PBC in the list of maintainers.